src/core/math_func.hpp

```diff
@@ -83,6 +83,29 @@ static inline T Clamp(const T a, const T min, const T max)
 	return a;
 }
 
+/**
+ * Clamp a value between an interval.
+ *
+ * This function returns a value which is between the given interval of
+ * min and max. If the given value is in this interval the value itself
+ * is returned otherwise the border of the interval is returned, according
+ * which side of the interval was 'left'.
+ *
+ * @note If the min value is greater than the max, return value is the average of the min and max.
```

Comment:
What's the rationale for this? Just "something is better than crashing"?

Reply:
Yup, exactly that! No reason to crash like this, as there is nothing actually wrong with it not fitting on the screen. So finding some value is better than saying: fuck this shit, I am out.

So yeah :)